### PR TITLE
Update changelog for v1.3.3

### PR DESCRIPTION
**Pull request by pschorf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
  
+## [1.3.3] - 2017-06-09
+### Added
+- Added simulator to test scheduler performance, from @wyegelwel
+### Changed
+- Fixed bug where job submit errors would return 201, from @pschorf
+- Optimizations in ranking to improve schedule time, from @shams
```
Comment (shamsimam) [inline CHANGELOG.md]: `shamsimam` please

Comment (pschorf) [inline CHANGELOG.md]: Done

File: integration/setup.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.0
+current_version = 1.3.4.dev1
```
Comment (sradack) [inline integration/setup.cfg]: Can we obtain this version automatically somehow?

File: integration/setup.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
 setup(
     name='cook_integration',
-    version='0.1.0',
+    version='1.3.4.dev1',
```
Comment (sradack) [inline integration/setup.py]: Can we obtain this version automatically somehow?

File: simulator/project.clj [hunk 4]
```diff
@@ -1,4 +1,4 @@
-(defproject cook/sim "0.1.0-SNAPSHOT"
+(defproject cook/sim "1.3.4-SNAPSHOT"
```
Comment (sradack) [inline simulator/project.clj]: Should this version change?

File: travis/prepare.sh [hunk 5]
```diff
@@ -7,4 +7,4 @@ lein uberjar
 
 cd  $PROJECT_DIR/../travis
 unzip datomic-free-0.9.5394.zip
-cp $PROJECT_DIR/../scheduler/target/cook-1.0.1-SNAPSHOT.jar datomic-free-0.9.5394/lib/
+cp $PROJECT_DIR/../scheduler/target/cook-1.3.4-SNAPSHOT.jar datomic-free-0.9.5394/lib/
```
Comment (sradack) [inline travis/prepare.sh]: Can we obtain this version automatically somehow?

**Comment by pschorf** [comment]
I'm waiting for #363 to go in before I cut the release

File: integration/setup.cfg (hunk 2 above)
Comment (pschorf) [inline integration/setup.cfg]: I reverted this, I don't think it should change.

File: integration/setup.py (hunk 3 above)
Comment (pschorf) [inline integration/setup.py]: Reverted

File: simulator/project.clj (hunk 4 above)
Comment (pschorf) [inline simulator/project.clj]: Reverted

File: travis/prepare.sh (hunk 5 above)
Comment (pschorf) [inline travis/prepare.sh]: Done

**Comment by pschorf** [comment]
@wyegelwel updated with job constraints

**Comment by pschorf** [comment]
Rebased onto latest master, @wyegelwel can you take another look and merge?

**Pull request merged**